### ISSUE TITLE
Could you support the jxl (JPEG XL) format?

### ISSUE THREAD

**Issue by 2eronia** [Snipaste/feedback#3077]
### Feature Description

When Snipaste's image quality is set to -1 and the format is webp, the screenshots lose a lot of details.

If I set the image quality to 90, the file size of the screenshots resembles Pingo ( https://css-ig.net/ )
`\pingo -s4 -webp losslessPNGScreenshot.png`.
compressed webp
or libjxl ( https://github.com/libjxl/libjxl )
`cjxl -e 10 losslessPNGScreenshot.png visuallylosslessJXL.jxl`.
compressed jxl
but still loses some details.

The webp and jxl shown above, their image detail is very close to the original lossless png.

I found that a screenshot application called Spectable on Linux supports the .jxl file format.
JXL has a higher compression ratio and better image quality than webp.
If I keep Spectable's default image quality at 90, I compare the jxl and webp that Spectable takes.
The image from the jxl file retains a lot of detail and is very close to the original image.
WebP loses a lot of detail and is larger than jxl.

It would be very appreciated if you could support jxl format.

**Comment by Deleted-Account** [comment]
Yep. JPEGXL renders a high compression ratio with a lot of detail.
Your rival PinPix is supposed to support the jxl format in the near future.
